lib/Dialect/TritonGPU/IR/LinearLayoutConversions.cpp

```diff
@@ -1435,8 +1475,10 @@ chooseDsReadTrLayout(Attribute enc, ArrayRef<int64_t> shape,
 }
 
 LinearLayout chooseScaledWmmaScaleLayout(MLIRContext *ctx, int dotOperandIdx,
-                                         ArrayRef<unsigned> warpsPerCTA,
-                                         ArrayRef<int64_t> dotOperandShape) {
+                                         ArrayRef<int64_t> dotOperandShape,
+                                         unsigned wmmaMDim,
+                                         ArrayRef<unsigned> tilesPerWarp,
```

Comment:
What is the context of this parameter? I get the feeling that is very specific and we might be able to infer it.